feat: show running builds in the Debug page (#25936)
cypress-io/cypress: cli/CHANGELOG.md

**Features:**

- It is now possible to control the number of connection attempts to the browser using the CYPRESS_CONNECT_RETRY_THRESHOLD Environment Variable. Learn more [here](https://docs.cypress.io/guides/references/advanced-installation#Environment-variables). Addressed in [#25848](https://github.com/cypress-io/cypress/pull/25848).

> - The Debug page is now able to show real-time results from runs that are still running in Cypress Cloud.  Addresses [#25759](https://github.com/cypress-io/cypress/issues/25759).

Reviewer: ```suggestion
- The Debug page is now able to show real-time results from in-progress runs.  Addresses [#25759](https://github.com/cypress-io/cypress/issues/25759).
```

Using "run" twice feels a bit awkward, just a wording suggestion
